raspberry_model.py

```diff
@@ -0,0 +1,69 @@
+from ner_processing.master_mapping import DATA_IDS
+from ner_processing.master_mapping import MESSAGE_IDS
+from ner_processing.message import Message
+from typing import Optional
+import can
+import os
+
+
+class RaspberryModel:
```

Comment:
We should rename this to something other than Raspberry model. The data we are trying to enrapture is data relevant to the car, not the Raspberry pi itself. You might even want to consider splitting the model into multiple components of the car, like having a BMS model, motor controller model, MPU model, TCU model, etc.

Reply:
Also I would 100% be down for refactoring the backend into a faster and more low level language like rust/C++. Integration of a unified CAN decoding library that we can use cross platform would be nice too